catalog/core/catalog-core-versioning/versioning-common/src/main/java/ddf/catalog/core/versioning/MetacardVersion.java

```diff
@@ -137,11 +148,16 @@ public static Action ofMetacard(Metacard metacard) {
 
     public static final String VERSION_TAGS = prefix.apply("tags");
 
+    public static final String VERSION_TYPE = prefix.apply("type");
+
+    public static final String VERSION_TYPE_BINARY = prefix.apply("type-binary");
+
     private static MetacardType metacardVersion;
 
+    private static HashSet<AttributeDescriptor> versionDescriptors =
```

Comment:
I know it's only a private method, but still should prefer using interfaces on signatures. `Set<AttributeDescriptor>`.